reconcile/acs_policies.py

```diff
@@ -0,0 +1,230 @@
+            if gql_policy.notifiers
+            else [],
+            severity=f"{gql_policy.severity.upper()}_SEVERITY",  # align with acs api severity value format
+            scope=sorted(
+                [
+                    Scope(cluster=cs.name, namespace="")
+                    for cs in cast(
+                        gql_acs_policies.AcsPolicyScopeClusterV1,
+                        gql_policy.scope,
+                    ).clusters
+                ],
+                key=lambda s: s.cluster,
+            )
+            if gql_policy.scope.level == "cluster"
+            else sorted(
+                [
+                    Scope(cluster=ns.cluster.name, namespace=ns.name)
+                    for ns in cast(
+                        gql_acs_policies.AcsPolicyScopeNamespaceV1,
+                        gql_policy.scope,
+                    ).namespaces
+                ],
+                key=lambda s: s.cluster,
+            ),
+            categories=sorted([POLICY_CATEGORIES[pc] for pc in gql_policy.categories]),
+            conditions=conditions,
+        )
+
+    def _build_policy_condition(
+        self, condition: AcsPolicyConditionsV1
+    ) -> PolicyCondition | None:
+        field_name = POLICY_CONDITION_FIELD_NAMES[condition.policy_field]
+        match condition.policy_field:
+            case "cvss":
+                cvss_condition = cast(
+                    gql_acs_policies.AcsPolicyConditionsCvssV1, condition
+                )
+                return PolicyCondition(
+                    field_name=field_name,
+                    negate=False,
+                    values=[
+                        f"{POLICY_CONDITION_COMPARISONS[cvss_condition.comparison]}{cvss_condition.score}"
+                    ],
+                )
+            case "severity":
+                severity_condition = cast(
+                    gql_acs_policies.AcsPolicyConditionsSeverityV1, condition
+                )
+                return PolicyCondition(
+                    field_name=field_name,
+                    negate=False,
+                    values=[
+                        f"{POLICY_CONDITION_COMPARISONS[severity_condition.comparison]}{severity_condition.level.upper()}"
+                    ],
+                )
+            case "cve":
+                cve_condition = cast(
+                    gql_acs_policies.AcsPolicyConditionsCveV1, condition
+                )
+                return PolicyCondition(
+                    field_name=field_name,
+                    negate=False,
+                    values=[str(cve_condition.fixable).lower()],
+                )
+            case "image_tag":
+                image_tag_condition = cast(
+                    gql_acs_policies.AcsPolicyConditionsImageTagV1, condition
+                )
+                return PolicyCondition(
+                    field_name=field_name,
+                    negate=image_tag_condition.negate or False,
```

Comment:
Was wondering why this was the only use of negation but then I double checked with the schema: https://github.com/app-sre/qontract-schemas/pull/571/files#diff-6bad3cee90319b7494304e3ce4e439992becd03f9d8825eca7f44687f8184ee9R734

May be worth a comment for future reviewers of this code

Reply:
good call. [addressed](https://github.com/app-sre/qontract-reconcile/pull/3987/commits/e470dc59cd78a06f4fefb16d805e5039360c0fa2) to make the attribute `Optional` on the class for clarity and comment on current usage